src/utils/helpers/GraphHelpers.ts

```diff
@@ -334,10 +383,19 @@ export function findGroupBackgroundDimension(nodesGroup: DependencyNode[]) {
     let rightLimitNode = nodesGroup[0];
 
     nodesGroup.forEach((node: DependencyNode) => {
-        if (!node.x || !node.y || !rightLimitNode.x || !leftLimitNode.x || !upperLimitNode.y || !lowerLimitNode.y) {
+        if (
+            !node.x ||
```

Comment:
is it possible that fields from this ``if`` statement have values equal to ``0`` ? If yes, should we stop go further, to the following ifs ?

Reply:
Should this node even exist with those values empty ? If it should maybe have a helper function for this check, like isNodeEmpty

Reply:
Do You agree on me fixing this logic in another PR? It looks like a bigger change than extracting some logic to a function